pkg/ebpf/tracee.go

```diff
@@ -1076,3 +1091,17 @@ func (t *Tracee) invokeInitEvents() {
 		}
 	}
 }
+func (t *Tracee) getTracedIfaceIdx(ifaceName string) int {
+	return findInList(ifaceName, &t.config.Filter.NetFilter.InterfaceToTrace)
+}
+func (t *Tracee) getCapturedIfaceIdx(ifaceName string) int {
+	return findInList(ifaceName, &t.config.Capture.NetIfaces)
+}
+func findInList(element string, list *[]string) int {
+	for idx, name := range *list {
+		if name == element {
+			return idx
+		}
+	}
+	return -1
+}
```

Comment:
As we're in go and not in C, it would be better to return an error instead of -1.